site/domains.md

```diff
@@ -57,13 +57,33 @@ Perform these steps to prepare your Kubernetes cluster to run a WebLogic domain:
 Please be aware of the following important considerations for WebLogic domains
 running in Kubernetes.
 
-* Channel listen addresses in a configuration either must be left completely unset (for example, not set to anything), or must be set to the exact required value, which will be in the form of the `domainUID`
-  followed by a hyphen and then the server name (with all lower case, underscores converted to dashes).  For example `domain1-admin-server`. This includes default, SSL, admin, and custom channels.     
-* If you choose to expose any WebLogic channels outside the Kubernetes cluster, for example, the administration port or a T3 channel to
-  allow WLST access, you need to ensure that you allocate each channel a unique port number across the entire
-  Kubernetes cluster.  If you expose the administration port in each WebLogic domain in the Kubernetes cluster, then each one must
-  have a different port.  This is required because `NodePorts` are used to expose channels outside the Kubernetes cluster.
-* If using a `hostPath` persistent volume, then it must be available on all worker nodes in the cluster and have read/write/many permissions for all container/pods in the WebLogic Server deployment.  Be aware
+* _Domain Home Location:_ The WebLogic domain home location is determined by the domain resource 'domainHome' if set,
+  and otherwise a default location is determined by the `domainHomeInImage` setting. If a domain resource 'domainHome' field is not set
+  and `domainHomeInImage` is `true` (the default), the operator will
+  assume the domain home is a directory under `/u01/oracle/user_projects/domains/` and report an error if no domain is found
+  or more than one domain is found.  If a domain resource 'domainHome' field is not set and `domainHomeInImage` is `false`, the operator will
+  assume the domain home is `/shared/domains/DOMAIN_UID`.
+* _Log File Locations:_ The operator can automatically override Weblogic domain and server log locations using situational
+  configuration overrides.  This occurs if the Domain resource `logHomeEnabled` field is explicitly set to `true`, or if `logHomeEnabled` isn't set 
```

Comment:
using situational configuration overrides -> using configuration overrides (globally) This is your call. IMO, the feature is called Configuration Overrides (or Configuration Overriding); situational configuration is an alternative, common usage term. If you think using the term situational configuration helps users to recognize this specific feature, then we should keep it. However, whenever you use the term, situational configuration, you should write it out completely, not abbreviate it as "situational config."